app/main/recipe_parser.py

```diff
@@ -171,6 +172,29 @@ def update_steps(self, file, steps):
         with open(file, 'w') as f:
             json.dump(updated_recipe, f, indent=4, sort_keys=True)
 
+def ZSeriesRecipeImport(recipe):
+    r = {}
+    r['name'] = recipe['Name']
+    name = recipe['Name']
+    current_app.logger.debug(f'DEBUG: saving recipe {name}')
+    r['clean'] = False
+    r['id'] = recipe['ID']
+    r['start_water'] = recipe['StartWater']
+    r['steps'] = []
+    for step in recipe['Steps']:
+        s = {}
+        s['name'] = step['Name']
+        s['temperature'] = step['Temp']
+        s['step_time'] = step['Time']
+        s['location'] = step['Location']
+        s['drain_time'] = step['Drain']
+        r['steps'].append(s)
+    filename = zseries_recipe_path().joinpath('{}.json'.format(r['name'].replace(' ', '_')))

```

Comment:
Global - we should either update to f-Strings or keep format.. @tmack8001 ?

Reply:
sure let's do a full refactor and choose one pattern 🃏